### PR TITLE
[Tutorial] fix the link thing for the pass tutorial

### PR DESCRIPTION
**Pull request by were**

### CODE REVIEW

File: tutorials/dev/low_level_custom_pass.py [hunk 1]
```diff
@@ -7,18 +7,22 @@
...
+
  Prerequisites
--------------
+--------------
+
 Before reading this tutorial, we assume readers have already known these topics well:
+
 - Writing an algorithm in TVM and schedule it. Otherwise, see example tutorials like
-  `Optimize GeMM on CPU <https://docs.tvm.ai/tutorials/optimize/opt_gemm.html>_`.
+  `Optimize GeMM on CPU <https://docs.tvm.ai/tutorials/optimize/opt_gemm.html>`_.
```
Comment (tqchen) [inline tutorials/dev/low_level_custom_pass.py]: use reference to set the link

Comment (tqchen) [reply tutorials/dev/low_level_custom_pass.py]: restructured text ref

Comment (were) [reply tutorials/dev/low_level_custom_pass.py]: how can i use relative link?

Comment (were) [reply tutorials/dev/low_level_custom_pass.py]: cuz i do not know the environment of how it is built to html, i cannot write the rel link.

**Comment by were** [comment]
@tqchen I guess I fixed it?

**Pull request merged**